projects/ngrx-auto-entity/src/lib/actions/actions.ts

```diff
@@ -19,7 +20,7 @@ export class LoadSuccess<TModel> extends EntityAction<TModel> {
 }
 
 export class LoadFailure<TModel> extends EntityAction<TModel> {
-  constructor(type: new () => TModel, public error: any, correlationId?: string) {
+  constructor(type: new () => TModel, public error: IEntityError, correlationId?: string) {
```

Comment:
I don't think we can guarantee that all errors at this point will be IEntityError. We have to be able to handle any error that the developers may throw from within their entity services, which could be Http errors, or custom errors. The IEntityError is an internal model that wraps errors between the internal entity service implementation (which is what calls the developer entity services), and the failure actions. But the failure actions get an "unwrapped" raw error object: `IEntityError.err`

The actions already expose IEntityInfo, so we don't need to expose that again.

Reply:
gotcha, we can close for now.